website/docs/d/wafregional_web_acl.html

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Can you please add `.markdown` to the end of the file name here? Otherwise the website page does not load properly. You can verify this with `make website`.

![Screen Shot 2019-07-17 at 11 36 19 AM](https://user-images.githubusercontent.com/189114/61390794-70ebd380-a889-11e9-8c0b-07f56bfcb79b.png)